clusters/development/overlay/radix-platform/radix-cicd-canary/cron-scaledobject.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: cron-scaledobject
+  namespace: radix-cicd-canary
+spec:
+  scaleTargetRef:
+    name:
```

name must be specified
```suggestion
    name: radix-cicd-canary
```